android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt

```diff
@@ -0,0 +1,22 @@
+package com.onegini.mobile.sdk.flutter
+
+enum class OneWelcomeWrapperErrors(val code: Int, val message: String) {
+    GENERIC(8000, "Something went wrong"),
```

Comment:
reworked error names

Reply:
I don't think they all have to be suffixed with _ERROR, but they should be more descriptive of what they actually mean. It would make sense that the name and the message mean the same. Take for instance the following error.
```kotlin
SECURITY_CONTROLLER_ERROR(8030, "Security controller class not found"),
```
The _ERROR doesn't add information, but we could rename it to something like `SECURITY_CONTROLLER_NOT_FOUND`

The error `ONEWELCOME_SDK_NOT_INITIALIZED_ERROR` is now longer than it's message `"OneWelcomeSDK not initialized"`, which implies that the name could just be the same as the message.

Reply:
i like the error prefix, because that highlights the use of these values to be errors sdkError(.ONEWELCOME_SDK_NOT_INITIALIZED_ERROR) makes it more clear I think that we are dealing with error values compared to  ONEWELCOME_SDK_NOT_INITIALIZED, but if you hardly disagree I can remove it based on what you said

Reply:
updated